client.js

```diff
@@ -1,31 +1,47 @@
-function u8tohex (arr) {
-  var vals = [ '0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'a', 'b', 'c', 'd', 'e', 'f' ]
-  var ret = ''
-  for (var i = 0; i < arr.length; ++i) {
-    ret += vals[(arr[i] & 0xf0) / 0x10]
-    ret += vals[(arr[i] & 0x0f)]
+function MonetizerClient (opts) {
+  var domain = new URL(window.location).origin
+  this.url = domain
+  if (opts && opts.url) {
+    this.url = opts.url
   }
-  return ret
-}
+  this.cookieName = (opts && opts.cookieName) || '__monetizer'
+  this.receiverUrl = this.url + '/__monetizer/:id'
```

Comment:
if URL is set to something other than the origin, it will still append `/__monetizer/:id` to it. It should probably default to `window.location.origin + '/__monetizer/:id'` if `opts.url` isn't specified.

Reply:
This may be a rare scenario but what if the monetizing server is run elsewhere or the client wants to send money to another server? I think it's okay to default it to the origin (which we're doing already?)